harness/determined/core/_train.py

```diff
@@ -16,6 +19,27 @@ class EarlyExitReason(enum.Enum):
     USER_REQUESTED_STOP = "EXITED_REASON_USER_REQUESTED_STOP"
 
 
+class TensorboardSyncMode(enum.Enum):
```

Comment:
Need to add this class to the Core API reference docs

Reply:
done.